salesagility/SuiteCRM
# Feature/sentry

## Description



Allows SuiteCRM exception handling to utilize [Sentry](https://sentry.io).

## Motivation and Context

Issue reference: #7849 

## How To Test This

1. Register a new application on https://sentry.io
2. Set the following config values:

```
    'sentry' => [
        'enabled' => true,
        'dsn' => '*Get this from sentry',
    ],
```
3. Check that the SuiteCRM exceptions and error logging goes to sentry.

## Types of changes


### Final checklist
